packages/contracts/helpers/types.ts

```diff
@@ -76,6 +76,8 @@ export enum AavePools {
 }
 
 
+
```

Comment:
I recommend you set up a linter on typescript files. Once I implement strict CI guidelines, ill show u how to set that up